docs/api/apiv3/paths/work_schedule_non_working_days.yml

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
we usually put the path here as a comment, to make reviewing this file more easy.

Reply:
Thanks. Done.